package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/http",
-  "version": "1.4.1-ergon-1",
+  "version": "1.4.1-ergon-2",
```

Comment:
I would always run an `npm install` after changing the version so the `package-lock.json file gets updated too.